Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM alpine:3.8
+
+WORKDIR /app
+
+RUN apk add --no-cache curl
```

Comment:
Might want to add the [`ca-certificates`](https://pkgs.alpinelinux.org/package/v3.3/main/x86/ca-certificates) package too.

Reply:
You mean also add an env var for defining a path to self signed certs?

Reply:
No that is to install a list of "trusted" CAs. Like the usual suspects verisign, comodo, etc.

Reply:
Yup, it comes as a dependency of curl.